footer.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="/style.css" rel="stylesheet">
+    <link href="footer.css" rel="stylesheet">

```

Comment:
Vamos ter somente um arquivo de css, `style.css`, pra deixar o seu código de css separado, coloca `/* FOOTER */` no início do seu código de css. Pode renomear o `footer.css` para `style.css`.